src/components/AppNavigationToggle/AppNavigationToggle.vue

```diff
@@ -36,13 +37,18 @@
 
 <script>
 import Button from '../Button/index.js'
+import Tooltip from '../../directives/Tooltip/index.js'
 import { t } from '../../l10n.js'
 
 import Menu from 'vue-material-design-icons/Menu'
 
 export default {
 	name: 'AppNavigationToggle',
 
+	directives: {
+		tooltip: Tooltip,
+	},
```

Comment:
Is this needed? It's not being done in https://github.com/nextcloud/nextcloud-vue/blob/d806d43e6b70364e740d8a00d71df46b547b4a7a/src/components/AppContent/AppDetailsToggle.vue#L24

Reply:
Good question.

https://github.com/nextcloud/nextcloud-vue/blob/498364d197c5257d758a48853abf87f71ba1d8d2/src/components/Actions/Actions.vue#L248 does it like this. But I will check.

Reply:
Maybe the other component only works by accident as every importer does it by accident as they import the Tooltip component:
https://github.com/nextcloud/spreed/blob/master/src/main.js#L50

Reply:
I tried it with Tasks where we don't separately import the directive with `Vue.use()` and I get an error `[Vue warn]: Failed to resolve directive: tooltip` when I don't import it explicitly here. So, it seems to be needed.

And I guess we should actually also import it explicitly in https://github.com/nextcloud/nextcloud-vue/blob/d806d43e6b70364e740d8a00d71df46b547b4a7a/src/components/AppContent/AppDetailsToggle.vue#L24

Reply:
I fixed the `AppDetailsToggle` in https://github.com/nextcloud/nextcloud-vue/pull/2887